WordPress/PHPCSAliases.php

```diff
@@ -61,21 +61,23 @@ class_alias( 'PHP_CodeSniffer\Standards\Squiz\Sniffs\WhiteSpace\SemicolonSpacing
 	 * @link https://github.com/squizlabs/PHP_CodeSniffer/issues/1564
 	 * @link https://github.com/squizlabs/PHP_CodeSniffer/issues/1591
 	 */
-	spl_autoload_register( function ( $class ) {
-		// Only try & load our own classes.
-		if ( stripos( $class, 'WordPress' ) !== 0 ) {
-			return;
-		}
+	spl_autoload_register(
+		function ( $class ) {
```

Comment:
Anonymous functions don't need to be treated as a closure and assigned to a variable first? Becuase this would be an exception to the "parameter must only be on a single line" part of the rule.

Reply:
That will be dealt with in a separate sniff via a warning - see #1330.

This PR just activates the parts of the `PEAR.Functions.FunctionCallSignature` sniff which were disabled and are step 1 to fixing #1330.